### PR TITLE
Add/remove nft strings

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -55,7 +54,8 @@
             "nftTypes": {
                 "ipfs": "IPFS",
                 "notTrue": "Not True",
-                "trueNft": "True NFT"
+                "trueNft": "True NFT",
+                "tip4": "TIP-4"
```
Comment (sertony) [inline src/ko.json:58]: @kkkhesha can you translate, please?

Comment (sertony) [inline src/ko.json:58]: @kkkhesha can you translate, please?

File: src/ru.json [hunk 2]
```diff
@@ -56,7 +55,8 @@
             "nftTypes": {
                 "ipfs": "IPFS",
                 "notTrue": "Not True",
-                "trueNft": "True NFT"
+                "trueNft": "True NFT",
+                "tip4": "TIP-4"
```
Comment (sertony) [inline src/ru.json:59]: @AnnStepanova can you translate, please?

Comment (vetkamak) [reply src/ru.json:59]: @sertony Ann says we don't need to translate it)

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [reply src/ko.json:58]: It's better to leave in English

**Pull request merged**